proknow/Roles.py

```diff
@@ -39,51 +40,26 @@ def create(self, name, permissions):
             :class:`proknow.Exceptions.HttpError`: If the HTTP request generated an error.
 
         Example:
-            This example creates a new role called "Researchers" with full permissions on the
-            "Research" workspace::
+            This example creates a new role called "Researchers"::
```

Comment:
Roles are a collection of permissions so I think this should be something like "Renaming Rule Maintainer" or something like that. In other words, it doesn't make sense to name a role after the user or group to which it will be assigned.